app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,78 @@
+class TweetsController < ApplicationController
+
+  get "/tweets" do
+    @tweets = Tweet.all
+    if logged_in?
+      erb :"tweets/tweets"
+    else
+    redirect to "/login"
+    end
+  end
+
+
+  get "/tweets/new" do
+    if logged_in?
+    erb :"tweets/create_tweet"
+  else
+    redirect to "/login"
+  end
+end
+
+  post "/tweets" do
+    if !params[:content].empty?
```

Comment:
I think you could use [ActiveRecord validations](http://guides.rubyonrails.org/active_record_validations.html) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.